src/main/java/com/aws/greengrass/clientdevices/auth/connectivity/ConnectivityInformation.java

```diff
@@ -55,7 +58,7 @@ public ConnectivityInformation(DeviceConfiguration deviceConfiguration,
     }
 
     /**
-     * Get cached connectivity info.
+     * Get cached connectivity info. Items in this list are unique.
```

Comment:
how is this ensured?

Reply:
ConnectivityInfoCache returns a set, so the conversions in CISShadowMonitor would be extra.  Using set in CISShadowMonitor would show clearer intent,  wanted to have a small change w/o refactoring every usage to be set